NEWS.md

```diff
@@ -4,6 +4,9 @@
 * Introduces `REPLICATION_FACTOR` environment variable for configuring Kafka topic replication (MODINVSTOR-694)
 * Provides `instance-storage 7.7`
 
+* Defaults instance.previouslyHeld to false (MODINVSTOR-454)
+* Provides `instance-storage 7.7`
```

Comment:
Please move this to a separate section of the news for the current release rather than together with the last release